SLHCUpgradeSimulations/Geometry/test/phase2_digi_reco_pixelntuple_cfg.py

```diff
@@ -1,13 +1,8 @@
-# Auto generated configuration file
-# using: 
-# Revision: 1.19 
-# Source: /local/reps/CMSSW/CMSSW/Configuration/Applications/python/ConfigBuilder.py,v 
-# with command line options: step2 --conditions auto:phase2_realistic -s DIGI:pdigi_valid,L1,L1TrackTrigger,DIGI2RAW,HLT:@fake2,RAW2DIGI,L1Reco,RECO --datatier GEN-SIM-RECO -n 10 --geometry Extended2023D41 --era Phase2 --eventcontent FEVTDEBUGHLT --filein file:SingleMuPt1000_pythia8_cfi_GEN_SIM.root --runUnscheduled --no_exec
 import FWCore.ParameterSet.Config as cms
 
 
-from Configuration.Eras.Era_Phase2C9_cff import Phase2C9
-process = cms.Process('Phase2PixelNtuple',Phase2C9)
+from Configuration.Eras.Era_Phase2C11_cff import Phase2C11
```

Comment:
Is this change appropriate for `D49`?

Reply:
No, it isn't.

Reply:
Sorry for not catching this. I just pushed a fix